src/utils.jl

```diff
@@ -1128,3 +1128,50 @@ _argument_description(s::Symbol) =
 
 _document_argument(s::Symbol) =
     _fmt_paragraph(_argument_description(s), leadingspaces = 6 + length(string(s)))
+
+#
+# Computes the distance of the plot limit to a sample of points
+# equally spaced in the set and keeps the minimum distance found.
+#
+function _dmin_series(lim,x,y,nsamples)
+    dmin = typemax(eltype(lim))
+    step = max(1,div(length(x),nsamples))
+    for i in 1:min(nsamples,length(x))
+        isample = firstindex(x) + (i-1)*step
+        d = sum(abs2, lim .- (x[isample],y[isample]))
+        dmin = ifelse(d < dmin, d, dmin)
+    end
+    return dmin
+end
+
+#
+# This function guesses which is the best legend position by computing
+# the distance of uniform set of points of the data set to each of the 
+# edges of the plot. The edge that is found to more distant to the 
+# closest point is chosen as the best position.
+#
+function _guess_best_legend_position(lp::Symbol,plt;nsamples=50)
+    if lp != :best
+        return lp
+    end
```

Comment:
```suggestion
    lp === :best || return lp
```

Nitpick on concision.